src/components/notifications/AppExplorer/AppCard/index.tsx

```diff
@@ -35,7 +35,11 @@ const AppCard: React.FC<AppCardProps> = ({ name, description, logo, bgColor, url
 
   const { activeSubscriptions } = useContext(W3iContext)
 
-  const subscribed = activeSubscriptions.some(element => element.metadata.name === name)
+  useEffect(() => {
+    setSubscribing(false);
+  }, [userPubkey])
```

Comment:
Can we be more explicit about what's going on here ?
```suggestion
  useEffect(() => {
    if(userPubkey) setSubscribing(false);
  }, [userPubkey])
```

Reply:
No - `setSubscribing(false` should run even if `userPubKey` is falsy. I'll add a clarifying comment though